randovania/games/planets_zebeth/assets/hash_words.json

```diff
@@ -0,0 +1,65 @@
+[
+    "Acid",
+    "Atomic",
+    "Attack",
+    "Ball",
+    "Beam",
+    "Blue",
+    "Bomb",
+    "Boots",
+    "Brain",
+    "Brick",
+    "Bridge",
+    "Brinstar",
+    "Bubble",
+    "Dessgeega",
+    "Energy",
+    "Gamet",
+    "Geruta",
+    "Hi-Jump",
+    "Hive",
+    "Holtz",
+    "Hopper",
+    "Ice",
+    "Kraid",
+    "Long",
+    "Mella",
+    "Mellow",
+    "Memu",
+    "Metroid",
+    "Missile",
+    "Morph",
+    "Mother",
+    "Multiviola",
+    "Nest",
+    "Norfair",
+    "Nova",
+    "Pirate",
+    "Polyp",
+    "Ridley",
+    "Rinka",
+    "Rio",
+    "Ripper",
+    "Samus",
+    "Screw",
+    "Skree",
+    "Spark",
+    "Squeept",
+    "Statue",
+    "Tangle",
+    "Tank",
+    "Tourian",
+    "Turret",
+    "Varia",
+    "Vine",
+    "Wave",
+    "Waver",
+    "Worker",
+    "Yakchi",
```

Comment:
What is this supposed to be?

Reply:
Same as above

Reply:
Github just shows the context of the previous lines
Line in question is Yakchi.